app/views/govuk_publishing_components/components/docs/error_message.yml

```diff
@@ -15,9 +15,22 @@ examples:
     data:
       text: "Please enter your National Insurance Number"
       id: "error_id"
+  with_dir_attribute:
+    description: Error items are a common pattern where a collection of error is passed with each item having a text attribute of the error
```

Comment:
This description and the one on the `with_items_and_dir_attribute` example are the same, and don't describe what is being demonstrated.

Reply:
Oops, that was a bit of lazy copy and paste. Fixed now 👍